CONTRIBUTING.md

```diff
@@ -0,0 +1,221 @@
+# Contributing Guide
+
+## Welcome to Percona Server for MySQL!
+
+We're excited to have you join the Percona community and participate in keeping open-source open.
+
+You can contribute in one of the following ways:
+
+1. Submit a bug report or a feature
+   request in [ our public Jira ](https://jira.percona.com/projects/PS/issues).
+2. Submit a pull request (PR) with the code patch.
+3. Contribute to [the documentation](https://github.com/percona/psmysql-docs/blob/innovation-release/contributing.md)
+4. Reach us on our [ Forums ](https://forums.percona.com/c/mysql-mariadb)
+
+This document describes the workflow for submitting pull requests.
+By contributing, you agree to the [Percona Community code of conduct](https://percona.community/contribute/coc/). Thank
+you for helping us enhance Percona Server for MySQL.
+
+## Why Your Contribution Matters?
+
+Main driving force of open-source project is community. We, as developers team, highly appreciate any contributions.
+
+It may be advisable to initially submit bug fixes to the upstream project that will eventually become part of the next
+release of Percona. Alternatively, Percona provides expedited response times, transparent development processes, and
+streamlined contribution acceptance within a public repository.
+
+## Creating a Jira Issue
```

Comment:
This is confusing.
Earlier, we say that Submitting a Jira ticket is one of the ways 
Then we say 
"This document describes the workflow for submitting pull requests" 
Then we have "Creating a Jira issue"? Is that necessary for a pull request? If so, why is it a separate method of contributing?

Reply:
Please cherck a newer version